libs/core/langchain_core/tracers/log_stream.py

```diff
@@ -228,7 +226,7 @@ def send(self, *ops: Dict[str, Any]) -> bool:
         try:
             self.send_stream.send_nowait(RunLogPatch(*ops))
             return True
-        except (ClosedResourceError, BrokenResourceError):
+        except ClosedResourceError:
```

Comment:
I don't see this error being raised anywhere in the write path, only in the read path?

Reply:
ah yah, i'll just remove this for now. I suspect we'll need to add some error handling as we encounter some failures (e.g., loop closed etc)